tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/WorkspaceMigrationImpl.java

```diff
@@ -43,6 +43,8 @@ public void problemsFound( List<Problem> problems )
     {
         final IWorkspaceRoot ws = ResourcesPlugin.getWorkspace().getRoot();
 
+        clearExistMarker( ws );
```

Comment:
We don't need to clear *all* markers, only clear the marker for the specific folder/project that is now being migrated.  So perhaps we need to modify the MigrateProjectHandler to remove all migration items for that current project.  If the project currently has markers we should prompt the user that continuing the tool will remove existing markers.